test.html

```diff
@@ -6,13 +6,13 @@
   </head>
   <body>
     <script type = "text/javascript">
-    function doIt(val) {
-    document.getElementById("theResult").innerHTML = val;
-    }
+{
+document.getElementById("insert").innerHTML = "write me to the screen";}
```

Comment:
DOM needs to be parsed before scripts can access elements. Use `defer` attribute on `SCRIPT` tag.